test/ingress/ingress.go

```diff
@@ -38,33 +38,36 @@ const (
 func getHTTPClient(timeout time.Duration) *http.Client {
 	// Get this Root PEM from https://letsencrypt.org/docs/staging-environment/#root-certificate
 	letsEncryptStagingRootPEM := `-----BEGIN CERTIFICATE-----
-MIIFATCCAumgAwIBAgIRAKc9ZKBASymy5TLOEp57N98wDQYJKoZIhvcNAQELBQAw
```

Comment:
Why is this needed?

Reply:
To talk to an end point that is using letsencrupt-staging certificate.

Reply:
https://letsencrypt.org/docs/staging-environment/#root-certificate

Reply:
But why is the _update_ needed? There is no issue reported for it, no PR description nor commit message, it's impossible to figure out what's going on :confused:

Reply:
There is something definitely wrong with the cert because the tests were failing:

```bash
ingress.go:108: got an HTTP error: Get "https://web-ui.ci1614080160-ip.test.lokomotive-k8s.net/": x509: certificate signed by unknown authority
```

Look at the 20 or so tests before this one: https://yard.lokomotive-k8s.net/teams/main/pipelines/pr-aws/jobs/test-lokomotive-aws-deployment/builds/1605

Reply:
I am merging this, since it blocking all PRs.